<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QTabWidget::pane { /* The tab widget frame */
     border-top: 2px solid #595fdd;
 }
  QTabWidget::tab-bar {
     alignment: center;
 }
 
QTabBar::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     border: 2px solid #595fdd;
     border-bottom-color: #595fdd; /* same as the pane color */
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     min-width: 8ex;
     padding: 2px;
 }
 
 QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #595fdd, stop: 0.4 #595fdd,
                                 stop: 0.5 #595fdd, stop: 1.0 #595fdd);
	color:white; 
}
 
 QTabBar::tab:selected {
     border-color: #9B9B9B;
     border-bottom-color: #595fdd; /* same as pane color */
 }
 
 QTabBar::tab:!selected {
     margin-top: 2px; /* make non-selected tabs look smaller */
 }
 
 /* make use of negative margins for overlapping tabs */
 QTabBar::tab:selected {
     /* expand/overlap to the left and right by 4px */
     margin-left: -4px;
     margin-right: -4px;
 }
 
 QTabBar::tab:first:selected {
     margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
 }
 
 QTabBar::tab:last:selected {
     margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
 }
 
 QTabBar::tab:only-one {
     margin: 0; /* if there is only one tab, we don't want overlapping margins */
}

 

QTableView {border: 3px solid #595fdd;text-align: top;padding: 4px;border-radius: 7px;border-bottom-left-radius: 7px;background: #fff;opacity:0.5;width: 15px;}
QTableView::item:focus{selection-background-color: #8FE585;}

#MyLoginForm {
background: red;
}

#mainFrame {
border: 3px solid red;
border-radius: 40px;
background: white;
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid #595fdd;
border-radius: 8px;
}
QTextEdit {
padding: 1px;
border-style: solid;
border: 2px solid #595fdd;
border-radius: 8px;
}
QDateEdit {
padding: 1px;
border-style: solid;
border: 2px solid #595fdd;
border-radius: 8px;
}
QTimeEdit {
padding: 1px;
border-style: solid;
border: 2px solid #595fdd;
border-radius: 8px;
}

QPushButton {
color: black;
  background: white;

border-width: 2px;
border-color: #595fdd;
border-style: solid;
border-radius: 10;
padding: 3px;
font-size: 12px;
padding-left: 5px;
padding-right: 5px;
min-width: 80px;
max-width:1800px;
min-height: 20px;
max-height: 20px;
}
QLabel {
font-weight: blue;
font-size: 13px;
}
QPushButton:hover { background-color: #595fdd; }
QComboBox{
border-style: solid;
border: 2px solid #595fdd;


 }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>952</width>
      <height>614</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ajouter_en">
     <property name="styleSheet">
      <string notr="true">#ajouter_en {
background-image: url(&quot;bg.png&quot;);
}</string>
     </property>
     <attribute name="title">
      <string>Ajouter Enfant</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_ajouterenfant">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>360</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nomenfant">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>124</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenomenfant">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>124</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ageenfant">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>124</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_parent">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>124</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom Enfant :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom Enfant :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Age Enfant :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Parent :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="afficher_en">
     <property name="styleSheet">
      <string notr="true">#afficher_en {
background-image: url(&quot;bg.png&quot;);
}</string>
     </property>
     <attribute name="title">
      <string>Afficher Enfant</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_enfant">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>621</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>470</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>470</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>470</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_triDate">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>470</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>911</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_13</zorder>
     <zorder>tableWidget_enfant</zorder>
     <zorder>lineEdit_6</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>pb_triDate</zorder>
    </widget>
    <widget class="QWidget" name="ajouter">
     <property name="styleSheet">
      <string notr="true">#ajouter {
background-image: url(&quot;bg.png&quot;);
}</string>
     </property>
     <attribute name="title">
      <string>Ajouter Activite</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_desc">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_datee">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>360</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>50</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom Activite :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Duree :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Temps:</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_dureeD">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_dureeA">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>118</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>160</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Datee :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>330</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Rdv :</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_temps">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_rdv">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_enfant">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>440</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Enfant :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="afficher">
     <property name="styleSheet">
      <string notr="true">#afficher {
background-image: url(&quot;bg.png&quot;);
}</string>
     </property>
     <attribute name="title">
      <string>Afficher Activite</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_activite">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>661</width>
        <height>351</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="actualiser_activite">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter  PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_trier_date">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par date</string>
      </property>
     </widget>
     <widget class="QPushButton" name="export_exl">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>440</y>
        <width>94</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>exporter EXL</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_activite">
      <property name="geometry">
       <rect>
        <x>735</x>
        <y>80</y>
        <width>151</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
